src/spyglass/lfp/v1/pipeline.py

```diff
@@ -0,0 +1,513 @@
+"""High-level functions for running the Spyglass LFP V1 pipeline."""
+
+from typing import Dict, Optional, Tuple
+
+import datajoint as dj
+
+# --- Spyglass Imports ---
+from spyglass.common import FirFilterParameters, IntervalList, Nwbfile, Raw
+from spyglass.lfp.analysis.v1 import LFPBandSelection, LFPBandV1
+from spyglass.lfp.lfp_electrode import LFPElectrodeGroup
+from spyglass.lfp.lfp_merge import LFPOutput
+from spyglass.lfp.v1 import LFPV1, LFPSelection
+from spyglass.position import PositionOutput  # Needed for ripple detection
+from spyglass.ripple.v1 import (
+    RippleLFPSelection,
+    RippleParameters,
+    RippleTimesV1,
+)
+from spyglass.utils import logger
+from spyglass.utils.dj_helper_fn import NonDaemonPool
+
+# --- Helper Function for Parallel Processing ---
+
+
+def _process_single_lfp_band(args_tuple: Tuple) -> Optional[Tuple]:
```

Comment:
Does `NonDaemonPool` require a tuple as opposed to just pulling items from kwargs?